client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -436,4 +445,43 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
             }
         });
     }
+
+    /**
+     * Checks if there are at least categories, motion-blocks or tags the user can select.
+     */
+    public checkMetaInfo(): void {
```

Comment:
Naming: This does check, but what does this method actually do: Updates the column definition for the 'state' column -> `updateStateColumnVisibility` or so..